src/components/YearbookSales/SalesBox/SalesBox.js

```diff
@@ -24,6 +35,11 @@ class SalesBox extends React.Component {
                     </div>
                 </div>
                 <div className="box-content">
+                    {!this.state.chartDataFetched && (
```

Comment:
I think you could export this for component new `Loader`. To component you could pass propertie isLoading. Thanks to you will can easy reuse it.